website/releases/2025-06-11-3.0.0-release.md

```diff
@@ -0,0 +1,59 @@
+---
+title: MLflow 3.0.0
+slug: 3.0.0
+authors: [mlflow-maintainers]
+---
+
+# MLflow 3.0
+
+The open source MLflow community has reached a major milestone. Today, we're releasing MLflow 3.0, which brings production-ready generative AI capabilities to the platform that millions of developers trust for ML operations.
+
+This isn't just another feature update. MLflow 3.0 fundamentally expands what's possible with open source ML tooling, addressing the observability and quality challenges that have made GenAI deployment feel like a leap of faith.
+
+## Major Updates
+
+### 🎯 MLflow `LoggedModel`
+
+MLflow 3.0 introduces a refined architecture with the new `LoggedModel` entity as a first-class citizen, moving beyond the traditional run-centric approach. This enables better organization and comparison of GenAI agents, deep learning checkpoints, and model variants across experiments.
+
+Learn more about [MLflow `LoggedModel`](https://mlflow.org/docs/latest/genai/data-model/#mlflow-loggedmodel-model-management) in the documentation.
+
+### 🔗 Strong Lineage Support
+
+Enhanced model tracking provides comprehensive lineage between models, runs, traces, prompts, and evaluation metrics. The new model-centric design allows you to group traces and metrics from interactive queries and automated evaluation jobs, enabling rich comparisons across model versions.
+
+### ⚡ Prompt Optimization
+
+The MLflow Prompt Registry now includes prompt optimization capabilities, allowing you to automatically improve prompts using evaluation feedback and labeled datasets. This includes versioning, tracking, and systematic prompt engineering workflows.
+
+Learn more about [prompt optimization](https://mlflow.org/docs/latest/genai/prompt-version-mgmt/prompt-registry/optimize-prompts) in the documentation.
+
+## Other Features
+
+**Streaming API for ResponsesAgent**: New streaming response capabilities through the ResponsesAgent class with predict_stream method, enabling real-time streaming for GenAI applications.
+
+**Auto-tracing support for Pydantic AI and `smolagents`**: Enhanced auto-tracing integrations for emerging GenAI frameworks, providing seamless observability out of the box.
+
+**Add `search_prompts` API for prompt registry**: New API functionality for searching and discovering prompts in the registry, making prompt management more efficient.
+
+**Support token tracking for OpenAI/LangChain auto-tracing**: Enhanced tracing now captures detailed token usage and cost information for better observability and cost management.
+
+**Record environment metadata in tracing**: Traces now capture comprehensive environment metadata including dependencies, system information, and execution context for better reproducibility.
+
+**UI support for video artifacts**: The MLflow UI now supports viewing video files directly in the artifact viewer, expanding beyond traditional ML artifacts to support multi-modal AI applications.
```

Comment:
Do we support this? 🤔

Reply:
Yes https://github.com/mlflow/mlflow/pull/15518